recipe/meta.yaml

```diff
@@ -15,7 +15,7 @@
 # https://github.com/pytorch/pytorch/blob/main/RELEASE.md#triton-dependency-for-the-release
 {% set triton = "3.1.0" %}
 
-# TODO Temporary pin, remove
+# TODO Temporary pin, remove me
 {% set mkl = "<2025" %}
```

Comment:
where was this hold over from?

Reply:
I think it was added while @Tobias-Fischer was trying to figure out why Windows builds were failing earlier.

Reply:
hmm. well we probably need to revert ;)

Reply:
Actually, I don't mind this for now. MKL 2025 has a lot of problems (or we have problems with it) - it's still not part of our blas infrastructure due to https://github.com/conda-forge/intel_repack-feedstock/issues/83

So as a measure to reduce the opportunities for the solver to get lost trying to (wrongly) update to mkl 2025, this is beneficial